.travis.yml

```diff
@@ -24,7 +24,7 @@ stages:
   - name: pip_package
     if: tag =~ ^v(\d+|\.)+[^a-z]\d+$
   - name: website_release
-    if: tag =~ ^v(\d+|\.)+[^a-z]\d+$ OR tag =~ ^v(\d+|\.)+[a-z]\d+$ OR tag = website
+    if: tag =~ ^v(\d+|\.)+[^a-z]\d+$ OR tag = website
```

Comment:
What is the effect of this change? What tags will no longer trigger a website_release?

Reply:
If I'm reading this regex correctly, this removes the ability to use lowercase letters in the build tags, which after reading through the previous tags seems like the wrong thing to remove. The first tag format allows for `v3.4.5A6` for example while the latter allows `v3.4.5a6`. I think I'm going to include both formats unless anyone sees an issue.